tests/xml/XMLBearTest.py

```diff
@@ -18,14 +20,76 @@
 <a>hey and hi</a>
 """.splitlines(keepends=True)
 
-InvalidXMLCorrectedTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_file,),
-                                        invalid_files=(invalid_xml_file,),
-                                        tempfile_kwargs={"suffix": ".xml"})
-
-InvalidXMLSyntaxTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_chars,),
-                                        invalid_files=(invalid_xml_chars,),
-                                        tempfile_kwargs={"suffix": ".xml"})
+dtd_file = os.path.join(os.path.dirname(__file__),
+                        "test_files",
+                        "note.dtd")
+
+schema_file = os.path.join(os.path.dirname(__file__),
+                           "test_files",
+                           "note.xsd")
+
+valid_xml_path = os.path.join(os.path.dirname(__file__),
+                              "test_files",
+                              "note.xml")
+
+valid_xml_url = os.path.join(os.path.dirname(__file__),
+                             "test_files",
+                             "concept-valid.xml")
+
+invalid_xml_schema = os.path.join(os.path.dirname(__file__),
+                                  "test_files",
+                                  "xsd-error.xml")
+
+invalid_xml_dtd = os.path.join(os.path.dirname(__file__),
+                               "test_files",
+                               "dtd-error.xml")
+
+invalid_xml_url = os.path.join(os.path.dirname(__file__),
+                               "test_files",
+                               "concept-invalid.xml")
+
+dtd_url = "http://docs.oasis-open.org/dita/v1.0.1/dtd/concept.dtd"
+
+InvalidXMLCorrectedTest = verify_local_bear(XMLBear,
+                                            valid_files=(valid_xml_file,),
+                                            invalid_files=(invalid_xml_file,),
+                                            tempfile_kwargs={"suffix": ".xml"})
+
+InvalidXMLSyntaxTest = verify_local_bear(XMLBear,
```

Comment:
You can merge this with the above one right ?
As there are no extra configs that need to be provided ?

(You can give 2 files in valid_files)

Not sure why they werent done together before